tacs/constitutive.pyx

```diff
@@ -215,6 +215,14 @@ cdef class MaterialProperties:
         cdef TacsScalar kappa2 = 230.0
         cdef TacsScalar kappa3 = 230.0
 
+        ISOTROPIC_KEYS = ["rho", "E", "nu", "G", "ys", "alpha", "specific_heat", "kappa"]
```

Comment:
Defining 'specific_heat' for orthotropic material would result in 'is_orthotropic=False' due to it being included in ISOTROPIC_KEYS.

Reply:
That's only the case if _only_ `specific_heat` is provided. If any keys are provided that aren't a part of this list, then it defaults to orthotropic (see [here](https://github.com/timryanb/tacs/blob/init_bug/tacs/constitutive.pyx#L221)).

For instance:
```
mat = MaterialProperties(specific_heat=0.5) # isotropic
mat = MaterialProperties(specific_heat=0.5, kappa1=0.2) # orthotropic
```

Reply:
My bad. I interpreted the if statement incorrectly. Thanks for clarifying.